### DISCUSSION TITLE
Island Top Stored In Database

### DISCUSSION THREAD

**Discussion by Justsean2** [BG-Software-LLC/SuperiorSkyblock2#749] [Denied Feature Requests] [answered]
Is it possible to store the island top players in the database? It would need to include the island leader's username, the amount of value they have, and their position on the leaderboard. This would be useful because then I could link it to my discord bot and I'd be able to make a sticky message with the island top rankings in discord.

Edit: I see that you have the discord SRV addon, however, I'd like this feature to be on my preexisting discord bot for my server so this would be the best way for me to implement the said feature.

Thank you for your time!
Sean

**Comment by OmerBenGera** [comment, marked as answer]
I am not gonna do such thing, it is a waste of storage and unused data to be stored. Not to mention that worth value isnt stored, and therefore it will require more coding and more waste.

You can implement it yourself using the API of the plugin, or even come with a better solution for that.
